### DISCUSSION TITLE
Displaying lists in React PDF

### DISCUSSION THREAD

**Discussion by Uken81** [diegomura/react-pdf#1359] [Q&A] [answered]
I'm wondering if React PDF is able to display an ordered/unordered list correctly in the downloaded PDF.
I have tried passing it in and it is displayed but not in a vertical list as I expected, instead its just mashes it all into a continuous single line.

Is it possible to pass in components with lists and have them appear as I wish?

**This is how the download appears for me**
<img width="587" alt="Screenshot 2021-06-03 112115" src="https://user-images.githubusercontent.com/64936770/120744382-30a9a300-c53e-11eb-8f4e-00854eb1938e.png">

I want it to appear like this:
hihihihhihhihihhihi
111111111111111
222222222222222

Any help would be greatly appreciated as I am really stuck on if I should continue to try and figure this out or move onto another library.

**Comment by diegomura** [comment, marked as answer]
Without looking at your JSX it's hard to say, but something like this should work:

```jsx
<View style={{ flexDirection: 'column' }}>
  <Text>hihihihhihhihihhihi</Text>
  <Text>111111111111111 </Text>
  <Text>222222222222222 </Text>
</View>
